src/applications/pre-need/tests/preneed-helpers.js

```diff
@@ -134,6 +134,7 @@ function completeBenefitSelection(client, data) {
       'input[name="root_application_claimant_desiredCemetery"]',
       data.claimant.desiredCemetery.label,
     )
+    .waitForElementVisible('#downshift-1-item-0', Timeouts.slow)
```

Comment:
Once this element shows up, we know the list of cemeteries has been pulled from the server and we can move on from the field